packages/server-wallet/src/wallet/constants.ts

```diff
@@ -0,0 +1 @@
+export const CHAIN_ID = '0x0';
```

Comment:
Why is this a constant?

Reply:
Eventually the `CHAIN_ID` will be compiled in or fetched at runtime, but for now I think it makes sense to point to a common constant.

Reply:
It may be good to put in the config file

Reply:
> I feel like it may be useful to make this a configuration option when instantiating a `Wallet`, e.g., `skipValidationCheck: true`

Good idea, did that here b3e11aa

Reply:
I think that probably makes sense but I think that can be outside the scope of this PR unless anyone has strong feelings about it.

Reply:
It looks like there is already a `config` class with ` chainNetworkId` so I should use that instead I think

Reply:
resolved 6d64209